src/hmem_cuda.c

```diff
@@ -301,6 +443,32 @@ int cuda_hmem_init(void)
 {
 	int ret;
 	int gdrcopy_ret;
+	bool cuda_enable_xfer;
+	int devices, dev_id, orig_dev_id;
+	void *ptr;
+	cudaIpcMemHandle_t handle;
+	cudaError_t err;
+
+
+	fi_param_define(NULL, "hmem_cuda_use_gdrcopy", FI_PARAM_BOOL,
+			"Use gdrcopy to copy data to/from CUDA GPU memory. "
+			"This requires an external library and kernel module "
+			"available at https://github.com/NVIDIA/gdrcopy. "
+			"Libfabric must be configured with gdrcopy support "
+			"using the --with-gdrcopy configure option. If gdrcopy "
+			"is not available or libfabric is not compiled with "
+			"gdrcopy support, this variable is not checked. "
+			"(default: true)");
+	fi_param_define(NULL, "hmem_cuda_enable_xfer", FI_PARAM_BOOL,
+			"Enable use of CUDA APIs for copying data to/from CUDA "
+			"GPU memory. This should be disabled if CUDA "
+			"operations on the default stream would result in a "
+			"deadlock in the application. Generally, this kind of "
+			"deadlock would occur when a CUDA kernel blocks until "
+			"some cudaMemcpy operation on the host completes "
+			"See 3.2.6.5.4: 'Implicit Synchronization' of the "
```

Comment:
I probably wouldn't refer to a specific section as this will likely go stale. I'd also suggest that these be trimmed down and this info is added into a man page somewhere.

Reply:
There is not currently any man page for HMEM. Should I add one, or append this information to a different man page?

Reply:
I'm fine with it landing in an existing man page if there's a good spot for it.